_source/_includes/setup/integrator-cli.md

```diff
@@ -0,0 +1,231 @@
+{%- if page.path contains '.adoc' -%}{% assign adoc = true %}{%- endif -%}
+{%- capture adminLink %}
+  {%- if adoc -%}https://developer.okta.com/login[Okta Admin Console]
+  {%- else -%}[Okta Admin Console](https://developer.okta.com/login)
+  {%- endif -%}
+{%- endcapture -%}
+
+{% if include.install != "false" %}
+  {%- if include.signup == "false" -%}Sign in to your {{ adminLink }}.
+  {%- else -%}Before you begin, you'll need a free Okta Integrator Free Plan account. Sign up at {{ adminLink }} to create a new account. If you already have an account, sign in to your {{ adminLink }}.
+  {%- endif -%}
+{% endif %}
+
+{%- if include.type == "spa" -%}
+  {%- assign parts = include.loginRedirectUri | split: '/callback' -%}
+  {%- assign baseUrl = parts[0] -%}
+{%- endif -%}
+
+{%- if include.type == "web" -%}
+  {%- assign parts = include.loginRedirectUri | split: '/authorization-code/callback' -%}
+  {%- assign baseUrl = parts[0] -%}
+{%- endif -%}
+
+{% if include.type == "jhipster" %}
+Then, in the Admin Console:
+
+1. Go to **Applications** > **Applications**
+2. Click **Create App Integration**
+3. Select **OIDC - OpenID Connect** as the sign-in method
+4. Select **Web Application** as the application type, then click **Next**
+5. Enter an app integration name or use the default
+  {% if include.loginRedirectUri and include.logoutRedirectUri %}6. Configure the redirect URIs:
+   - **Sign-in redirect URIs**: `{% if adoc %}\{% endif %}{{ include.loginRedirectUri }}`
+   - **Sign-out redirect URIs**: `{% if adoc %}\{% endif %}{{ include.logoutRedirectUri }}`
+  {% else %}6. Accept the default redirect URI values:
+   - **Sign-in redirect URIs**: `http://localhost:8080/login/oauth2/code/oidc` and `http://localhost:8761/login/oauth2/code/oidc`
+   - **Sign-out redirect URIs**: `http://localhost:8080` and `http://localhost:8761`
+  {% endif %}
+7. Click **Save**
+{% elsif include.type == "token" %}
```

Comment:
we should remove this section. We needed tokens before with Okta CLI, but we don't need to promote this poor practice.